api/v01/entities/vrp_input.rb

```diff
@@ -516,11 +516,15 @@ module VrpVehicles
     optional(:distance, type: Integer, desc: 'Maximum tour distance. Not available with periodic heuristic.')
     optional(:maximum_ride_time, type: Integer, desc: 'Maximum ride duration between two route activities')
     optional(:maximum_ride_distance, type: Integer, desc: 'Maximum ride distance between two route activities')
-    optional :skills, type: Array[Array[Symbol]],
+    optional :skills, type: Array[Array[Symbol]], default: [[]],
```

Comment:
Using default values within the active_hash model used to generate array with the same id for all objets of the problem. Could check if the object_id of skills obtained using this default value by two vehicles is different ?

Reply:
Now skills.object_id are different. Do you think it will be too costly to duplicate these ? In my opinion it is cleaner like this.